frontend/hub/collections/CollectionPage/CollectionImportLog.tsx

```diff
@@ -27,67 +28,134 @@ export function CollectionImportLog() {
         }&sort=-created&limit=1`
       : ''
   );
+  const ref = useRef<HTMLDivElement>(null);
 
   const { data: collectionImport } = useGet<CollectionImport>(
     collectionImportsResponse && collectionImportsResponse.data.length
       ? hubAPI`/_ui/v1/imports/collections/${collectionImportsResponse.data[0].id}/`
       : ''
   );
+
+  const StyledArrowDown = styled.div`
+    color: white;
```

Comment:
This could be moved outside the main component definition. Avoid to re-create the component every time the main one renders, and create all custom ones in the same place.

https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method

Reply:
Definetly, I wrote this before you commented on previous PR. I was going to redo this now :-)

Reply:
Definetly, I was going to redo this there as well.